editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -318,6 +325,8 @@ protected void languageChanged(String language, Composite eform, AbstractCamelMo
 		if (prop.getName().equalsIgnoreCase("expression")) {
 			// normal expression subnode - no cascading -> when.<expression>
 			// the content of expressionElement is the language node itself
+			LanguageModel model = getCamelModel(expressionElement).getLanguageModel();
+			Language lang = model.getLanguageByName(language);
```

Comment:
We should move this definition outside the if clauses so it can be accessed in the whole method. No need to create these variables in both if and else branches.